inc/namespace.php

```diff
@@ -125,6 +150,20 @@ function render_pat_settings_field() {
 	<?php
 }
 
+/**
+ * Display the input field for the GitHub personal acess token.
+ */
+function render_translate_settings_field() {
+	$translate = get_option( 'dc-translate' );
+
+	?>
+	<input type="checkbox" id="dc-translate" name="dc-translate" value="1" <?php checked(1, $translate, true); ?> />
```

Comment:
The checkbox looks weird to me for a settings page. I know there's other core settings that use checkboxes, but I think an `Enable`/`Disable` dropdown might look better.

Reply:
I think the same about checkboxes, but i we consider user inputs required to update the setting :
- checkbox : 1 click
- dropdown : 2 clicks

Maybe we could try to make the checkbox look less weird by juxtaposing the help text next to it ? Lite this:
![image](https://user-images.githubusercontent.com/22429540/192457971-4c4193df-3624-4386-af6d-f96727bd3f58.png)

Reply:
Fair enough. I agree, aligning the text to the right of the checkbox would be better than below, let's go with that.